build/source/dshare/ascii_util.f90

```diff
@@ -21,6 +21,7 @@
 module ascii_util_module
 USE nrtype
 implicit none
+integer(i4b),parameter,public :: linewidth=4096  ! max character number for one line
```

Comment:
Should this be the same length as in l.80 : `8192`?

Reply:
In some sense it's arbitrary.  It might be nice to have lengths like this be consistent, but I hadn't thought about it when implementing this.